File: 0201_0300/0276/0276.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, m = (int(x) for x in f.readline().split())
    q.write((str(n // m) + ' ') * (m - n % m) + (str(n // m + 1) + ' ') * (n % m))

File: 0201_0300/0276/0276.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, m;

	fscanf(f, "%d%d", &n, &m);
	for (int i = 0; i < m - n % m; ++i)
		fprintf(q, "%d ", n / m);
	for (int i = 0; i < n % m; ++i)
		fprintf(q, "%d ", n / m + 1);
	return 0;
}
